libs/handoversidesheet/src/lib/ui-sidesheet/HandoverSidesheet.tsx

```diff
@@ -250,6 +256,14 @@ const HandoverSidesheetComponent = (props: Required<HandoverProps>) => {
               Query <TabTitle data={queryPackages} isLoading={isDataFetchingQuery} />{' '}
             </Tabs.Tab>
 
+            <Tabs.Tab>
+              Notifications
+              <TabTitle
+                data={notifications}
+                isLoading={isDataFetchingNotifications}
+              />{' '}

```

Comment:
You seem to be getting the same formatter error as me 
```suggestion
              />
```